src/cmake/FindAPR-Util.cmake

```diff
@@ -48,5 +55,5 @@ else()
     endif()
 endif()
 
-find_package_handle_standard_args(APR-Util DEFAULT_MSG
+find_package_handle_standard_args(APR-Util
```

Comment:
How is removing `DEFAULT_MSG` related to this PR?

Reply:
Just so you can see the library being used - now the message output is consistent with FindAPR